addons/project-notes-tabs/userscript.js

```diff
@@ -0,0 +1,39 @@
+export default async function ({ addon, global, console }) {
+  while (true) {
+    let projectNotes = await addon.tab.waitForElement(".project-notes", { markAsSeen: true });
+    let allLabels = document.querySelectorAll(".project-textlabel").length;
+    for (let i = 0; i < allLabels; i++) {
+      document.querySelector(".project-textlabel").remove();
+    }
+    let tabs = projectNotes.insertBefore(
+      document.createElement("div"),
+      projectNotes.querySelector(".description-block")
+    );
+    tabs.classList.add("tabs-sa");
+    tabs.setAttribute("style", "margin-bottom: 0px;");
+    if (allLabels - 1) {
+      let intTab = tabs.appendChild(document.createElement("div"));
+      intTab.classList.add("tab-choice-sa");
+      let innerTab = intTab.appendChild(document.createElement("span"));
+      innerTab.innerText = "Instructions";
+      intTab.setAttribute("style", "margin-left: 5px;");
+    }
+    let notesTab = tabs.appendChild(document.createElement("div"));
+    notesTab.classList.add("tab-choice-sa");
+    let innerTab = notesTab.appendChild(document.createElement("span"));
+    innerTab.innerText = "Notes and Credits";
+    if (!(allLabels - 1)) notesTab.setAttribute("style", "margin-left: 5px;");
```

Comment:
It would be better if you used the `element.style` element, instead of setting the attribute (happens many times on this addon, and others by you). If two addons wanted to style the same element, only one would work.